src/Controls/src/Core/Platform/Android/DragAndDropGestureHandler.cs

```diff
@@ -272,10 +272,11 @@ public void OnLongPress(MotionEvent e)
 
 				// TODO MAUI
 				string clipDescription = String.Empty;//AutomationPropertiesProvider.ConcatenateNameAndHelpText(element) ?? String.Empty;
+				ClipData data = null;
 				ClipData.Item item = null;
 				List<string> mimeTypes = new List<string>();
 
-				if (!args.Handled)
+				if (args.PlatformArgs.ClipData is null)
```

Comment:
This is very different. The old way used the handled, now we use clip data. I assume once the user got the drag event, they would set this to true and it would skip the item detection - or part of it. Now it always does it? Should we rather use the clip data, and if not provided follow the same check and path? Is there an issue or a need for this change?

Reply:
Moved back!